Storage/api.h

```diff
@@ -219,6 +219,13 @@ STORAGE_HANDLERS_MAP_ENTRY GenerateRESTfulHandler(STORAGE& storage, const std::s
 }
 
 }  // namespace impl
+template <bool CALL>
```

Comment:
This really needs to be a general utility. I usually create a collection of files in a directory std_ext, where I add things like this. It would then take a syntax like
if_true<CONDITIONAL>::call(FUNCTION_LIKE);
if_false<CONDITIONAL> becomes a type alias.

Note that I actually have the boolean logic reversed for this case, because I already had the ExcludeTypeFromPersistence defined. TODO: make it make sense.